modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/sweep/sweep.h

```diff
@@ -33,14 +38,18 @@ struct Task
   bool completed_ = false;
 };
 
-class AAH_ASynchronousCommunicator;
-class AngleSet;
-class AngleSetGroup;
-class AngleAggregation;
-
-class SweepChunk;
+/**Contains the sweep plane data.*/
+struct SPLS
```

Comment:
Just a question: Should this be SPLD as in `S`weep `PL`ane `D`ata or does the trailing `S` stands for something else?

Reply:
SPLS -> Sweep Plane Local Subgrid

Reply:
That seems like that should be in the comment then?